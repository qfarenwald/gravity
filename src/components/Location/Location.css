.Location {
  color: white;
}

.Location-h4 {
  font-weight: lighter;
  letter-spacing: 25px;
  margin-top: 100px;
}

.Location-btn {
  background-color: #38B449;
  border: none;
  font-size: 1rem;
  letter-spacing: 5px;
  padding: 20px;
  margin: 50px;
  width: 200px;
}

.Location-btn:hover {
  background-color: #33993D;
}
