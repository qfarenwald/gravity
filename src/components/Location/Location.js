import React from 'react';
import { Link } from 'react-router-dom';
import './Location.css';

export const Location = () => {
  return (
    <section className="Location">
      <h4 className="Location-h4">CHOOSE LOCATION</h4>
      <Link to ="/trails"><button className="Location-btn">DENVER</button></Link>
      <Link to ="/trails"><button className="Location-btn">BOULDER</button></Link>
    </section>
  )
}
