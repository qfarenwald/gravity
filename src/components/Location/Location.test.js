import React from 'react';
import ReactDOM from 'react-dom';
import { Location } from './Location';
import { shallow } from 'enzyme';

describe('Location', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Location />)

    expect(wrapper).toMatchSnapshot()
  })
})
