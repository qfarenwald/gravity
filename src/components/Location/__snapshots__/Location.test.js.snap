// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Location should match snapshot 1`] = `
<section
  className="Location"
>
  <h3>
    CHOOSE A LOCATION
  </h3>
  <Link
    to="/trails"
  >
    <button
      className="Location-btn"
      onClick={[Function]}
    >
      DENVER
    </button>
  </Link>
  <Link
    to="/trails"
  >
    <button
      className="Location-btn"
      onClick={[Function]}
    >
      BOULDER
    </button>
  </Link>
</section>
`;
